config/config.go

```diff
@@ -0,0 +1,25 @@
+package config
+
+import "os"
+
+type userDbConfig struct {
+  User string
+  Password string
+}
+
+var config *userDbConfig
+
+func GetUserDbConfig() *userDbConfig {
```

Comment:
What is the rationale behind requiring distinct configurations for both the user and message?